src/frontend/src/components/authenticateBox.ts

```diff
@@ -18,51 +19,53 @@ import {
 import { displayError } from "./displayError";
 import { useRecovery } from "../flows/recovery/useRecovery";
 import { registerIfAllowed } from "../utils/registerAllowedCheck";
-import { withRef } from "../utils/lit-html";
 
-/** Properties for the authentication box component which authenticates a user
- * to II or to another dapp */
-type PageProps = { templates: AuthTemplates } & {
-  onContinue: (userNumber: bigint) => void;
-  recoverAnchor: (userNumber?: bigint) => void;
-  register: () => void;
-  addDevice: () => void;
-  anchors: bigint[];
-};
-
-/** Text and content to display in the box */
-export type AuthTemplates = {
-  message: TemplateResult;
-  button: string;
-  chasm?: ChasmOpts;
-};
-
-/** Options to display a "chasm" in the authbox */
-type ChasmOpts = {
-  info: string;
-  message: TemplateResult;
+/** Template used for rendering specific authentication screens. See `authnPages` below
+ * for meaning of "new", "useExisting" and "pick". */
+export type AuthnTemplates = {
+  new: {
```

Comment:
`new` sounds a bit like a reserved keyword to avoid, can you use something else?